src/pages/hegel/reference/mechanical-object/section-3.mdx

```diff
@@ -1,22 +1,36 @@
+>“Here, then, we have the manifest *contradiction* between the complete mutual *indifference* of the objects and the *identity* of their *determinateness*, or the contradiction of their complete *externality* in the *identity* of their determinateness.” (Hegel, 1991, 713)
 
->“This contradiction is, therefore, the *negative unity* of a number of objects which, in that unity, simply repel one another: this is the *mechanical process*.” (713)
+To return to my example of the rock that splashes into the water. From a mechanistic perspective, it makes not a bit of difference whether it is a rock or a boulder that splashed into the water because the determinateness of the mechanical objects is identical. This identity of determinateness has the effect of treating the mechanical objects as interchangeable because, with regard to their external determinateness, there is no distinction between one or the other. However, this is to ignore another equally crucial determination of the mechanical objects: their indifference. How can two objects be treated as identical whilst being indifferent to each other? Hegel’s response is that that is exactly where the logic of `Mechanism` leads us – towards the contradiction that the determinateness of a `mechanical object` is both identical to every other `mechanical object` whilst the object itself being indifferent to that very identical determinateness. This contradiction, then, is comprehended and grasped as the negative unity of the mechanical objects:
 
-The conclusion of the “A. The Mechanical Object”, then, is that these mechanical objects that were initially taken as immediately identical, indeterminate, and indifferent, are in fact in a contradiction with each other. This contradiction is a negative unity whereby what it is for a mechanical object to be itself is for its determinateness to be identical to the determinateness of other mechanical objects (the *unity* of the negative unity), whilst for it to be indifferent to other mechanical objects (the *negative* in negative unity. This new unity takes us into the next chapter of Mechanism, “B. The Mechanical Process”. 
+>“This contradiction is, therefore, the *negative unity* of a number of objects which, in that unity, simply repel one another: this is the *mechanical process*.” (Hegel, 1991, 713)
 
+The conclusion of the `A. The Mechanical Object`, then, is that these mechanical objects that were initially taken as immediately identical, indeterminate, and indifferent, are in fact in a contradiction with each other. This contradiction is a negative unity whereby what it is for a `mechanical object` to be itself is for its determinateness to be identical to the determinateness of other mechanical objects (the *unity* of the negative unity), whilst for it to be indifferent to other mechanical objects (the *negative* in negative unity. This new unity takes us into the next chapter of `Mechanism`, `B. The Mechanical Process`. 
```

Comment:
Also, I don't think you should be using back ticks for section titles, as these aren't technical terms per say but textual. What do you think?

Reply:
I think for textual headings we should just capitalize it normally without the backticks?